packages/app/services/actors.js

```diff
@@ -65,7 +71,18 @@ module.exports = {
                 'interop:applicationName': this.settings.app.name,
                 'interop:applicationDescription': this.settings.app.description,
                 'interop:applicationAuthor': this.settings.app.author,
-                'interop:applicationThumbnail': this.settings.app.thumbnail
+                'interop:applicationThumbnail': this.settings.app.thumbnail,
+                'oidc:client_name': this.settings.app.name,
+                'oidc:redirect_uris': this.settings.oidc.redirectUris,
+                'oidc:post_logout_redirect_uris': this.settings.oidc.postLogoutRedirectUris,
+                'oidc:client_uri': this.settings.oidc.clientUri,
+                'oidc:logo_uri': this.settings.app.thumbnail,
+                'oidc:tos_uri': this.settings.oidc.tosUri,
+                'oidc:scope': 'openid profile offline_access webid',
+                'oidc:grant_types': ['refresh_token', 'authorization_code'],
+                'oidc:response_types': ['code'],
+                'oidc:default_max_age': 3600,
+                'oidc:require_auth_time': true
```

Comment:
To the list, I would add:
```json
  "token_endpoint_auth_method": "none", <- to comply with oidc spec (where default is `client_secret_basic`)
  "policy_uri": "",
  "contacts": [],
  "application_type": "web",
```

Also, maybe it's better to leave out the prefixes, to improve compatibility with services that doen't speak jsonld? And adding `https://www.w3.org/ns/solid/oidc-context.jsonld` explicitly.

See also:
https://solidproject.org/TR/oidc#clientids-document
https://solid-client-identifier-helper.vercel.app/documentation

Reply:
> To the list I would add: 
> ```json
>   "token_endpoint_auth_method": "none", <- to comply with oidc spec (where default is `client_secret_basic`)
>   "policy_uri": "",
>   "contacts": [],
>   "application_type": "web",
> ```

We cannot have empty strings or arrays when we use a triple store for storage.
Adding `token_endpoint_auth_method` is a good idea, thanks.
For `application_type`, is it really "web" since this is a backend client ID ?

> Also, maybe it's better to leave out the prefixes, to improve compatibility with services that doen't speak jsonld? And adding https://www.w3.org/ns/solid/oidc-context.jsonld explicitly.

That's the whole problem of handling contexts correctly, that I haven't tackled yet.
Unfortunately https://www.w3.org/ns/solid/oidc-context.jsonld is a terrible context, with all @id being changed to `client_id` and entering into conflict with the AS context. So it cannot be used in the whole code base.
I've looked at the CSS code and, if it doesn't find the https://www.w3.org/ns/solid/oidc-context.jsonld context, it will parse the data as regular RDF and it will work.

Reply:
> We cannot have empty strings or arrays when we use a triple store for storage.

My bad, that was suppoed to be `this.settings.oidc.plicy_uri` and so on :)

>  For application_type, is it really "web" since this is a backend client ID ?

It doesn't have to be but it defaults to `web`, if not specified.

> That's the whole problem of handling contexts correctly, that I haven't tackled yet.
Unfortunately https://www.w3.org/ns/solid/oidc-context.jsonld is a terrible context, with all @id being changed to client_id and entering into conflict with the AS context. So it cannot be used in the whole code base.
I've looked at the CSS code and, if it doesn't find the https://www.w3.org/ns/solid/oidc-context.jsonld context, it will parse the data as regular RDF and it will work.

Mh, that's annoying. Would it make sense though for client identifier documents, go with both anyways and make use of jsonld context overriding?

Reply:
> For information, when fetching the document from the OIDC provider, we use the JsonLdContext header with https://www.w3.org/ns/solid/oidc-context.jsonld to have nicely formatted data.

Ah ok, I think that was the information missing for me, thanks! :)
How is it formatted, if no `JsonLdContext` header is passed though?

Reply:
It is correct JSON-LD data, but it will not be understood by a software that only reads JSON (as mentionned elsewhere, the Community Solid Server is able to understand JSON-LD no matter the context used).